### ISSUE TITLE
leading LF(\n) and CR (\r) characters produces a diff

### ISSUE THREAD

**Issue by fsoylemez** [javers/javers#531]
Hello,
When i compute diff of two strings(map values),i see that leading empty characters produce a diff.
is it the way that it should work,or is this a bug?

**Comment by bartoszwalacik** [comment]
That's the way String.equals() works

**Comment by fsoylemez** [comment]
just saying,would it be wise to trim() before calling equals()?

**Comment by bartoszwalacik** [comment]
Maybe as an option

**Comment by bartoszwalacik** [comment]
fixed in release 3.3.0, see https://javers.org/release-notes/

for example:

```java
 javersBuilder.registerValue(BigDecimal.class, (a,b) -> a.compareTo(b) == 0);

```
